sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/BroadcastExchangeExec.scala

```diff
@@ -141,7 +141,8 @@ case class BroadcastExchangeExec(
             longMetric("dataSize") += dataSize
             if (dataSize >= MAX_BROADCAST_TABLE_BYTES) {
               throw new SparkException(
-                s"Cannot broadcast the table that is larger than 8GB: ${dataSize >> 30} GB")
+                s"Cannot broadcast the table that is larger than" +
+                  s" ${MAX_BROADCAST_TABLE_BYTES >> 30}GB: ${dataSize >> 30} GB")
```

Comment:
sorry why `8GB` is not accurate? `BroadcastExchangeExec.MAX_BROADCAST_TABLE_BYTES` seems to be set statically as `8L << 30`, right?

Reply:
It's not currently inaccurate, but if that constant changed values, it would be